Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -156,21 +156,21 @@ def self.determine_bump_subject(old_contents, new_contents, subject_path, reason
 
   # Cherry picks a single commit that modifies a single file.
   # Potentially rewords this commit using {determine_bump_subject}.
-  def self.reword_package_commit(commit, file, reason: "", verbose: false, resolve: false, path: ".")
```

Comment:
I've renamed `path:` to `git_repo:` and removed the default value (it was never valid bc it was a `String`). Same for `squash_package_commits` below.